### PR TITLE
changed page design

### PR DESCRIPTION
**Pull request by LIUBOMYR-HOLKO**

### CODE REVIEW

**Review by XashDM — APPROVED** [review]
Look ok

File: sports_hub/sports_hub/wwwroot/css/AdminPage/languages.css [hunk 1]
```diff
@@ -182,8 +151,10 @@ tr {
 }
 
 #languageContainer {
-    margin-left: 20%;
-    margin-top: 5%
+    margin-left: 10%;
+    margin-right: 10%;
+    text-align: center;
+    margin-top: 5%;
```
Comment (SerhiyToroniy) [inline sports_hub/sports_hub/wwwroot/css/AdminPage/languages.css:157]: You can simplify this code - margin: 5% 10% 0px 10%;

File: sports_hub/sports_hub/wwwroot/css/AdminPage/languages.css [hunk 2]
```diff
@@ -264,4 +234,179 @@ label{
...
+.third-column {
+    padding-left: 51px;
+}
+
+.menu-page {
+    width: 603px;
+    height: 55px;
+    padding-left: 35px;
+    padding-top: 15px;
+    padding-right: 35px;
```
Comment (SerhiyToroniy) [inline sports_hub/sports_hub/wwwroot/css/AdminPage/languages.css:367]: The same code here (simplify it as well as margin above).

**Review by XashDM — APPROVED** [review]
Approved

**Pull request merged**